script.py

```diff
@@ -17,6 +19,21 @@
 AZURE_OPENAI_DEPLOYMENT_NAME_CHAT = os.getenv('AZURE_OPENAI_DEPLOYMENT_NAME_CHAT')
 AZURE_OPENAI_API_VERSION_CHAT = os.getenv("AZURE_OPENAI_API_VERSION_CHAT")
 
+# OpenAI configuration
+OPENAI_API_KEY = os.getenv('OPENAI_API_KEY')
+OPENAI_ENDPOINT = os.getenv('OPENAI_ENDPOINT')
+OPENAI_MODEL = os.getenv('OPENAI_MODEL')
+OPENAI_MODEL_CHAT = os.getenv('OPENAI_MODEL_CHAT')
+
+# Groq OpenAI configuration
+GROQ_OPENAI_API_KEY = os.getenv('GROQ_OPENAI_API_KEY')
+GROQ_OPENAI_ENDPOINT = os.getenv('GROQ_OPENAI_ENDPOINT')
```

Comment:
what are the values for endpoint, could you please enter examples in .env.example